Multisig/Data/Services/Safe Client Gateway Service/BalancesRequest.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 struct BalancesRequest: JSONRequest {
     private let safeAddress: String
-    private let networkId: Int
+    private let networkId: String
```

Comment:
Would it be more explicit to use UInt256String here?

Reply:
and in other places

Reply:
This is same as using the String to represent safeAddress instead of Address in the above line

Reply:
Yes, probably this is for the refactoring ticket that Dima mentioned